### ISSUE TITLE
False negative with :not

### ISSUE THREAD

**Issue by amannn** [atfzl/eslint-plugin-css-modules#48]
SCSS:
```scss
.root {
  &:not(.selected) {
    opacity: 0.5;
  }
}
```

JS:
```jsx
<div className={cx(styles.root, styles.selected)} />
```

**Comment by yuhsianw** [comment]
This should not be identified as a bug. In the example above, there're no selectors that target `.selected` in the SCSS file. `:not(.selected)` targets anything other then `.selected` so the error is semantically correct (i.e. it's undefined).

You should able to avoid this error by using sting literals for classes that aren't defined in the CSS module (syntax unchecked):
```JavaScript
<div className={cx(styles.root, "selected")} />
```
